backend/src/test/java/com/woowacourse/f12/application/auth/token/RefreshTokenRepositoryImplTest.java

```diff
@@ -0,0 +1,113 @@
+package com.woowacourse.f12.application.auth.token;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import com.woowacourse.f12.exception.unauthorized.DuplicatedRefreshTokenSavedException;
+import com.woowacourse.f12.exception.unauthorized.RefreshTokenNotFoundException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.context.jdbc.Sql;
+
+@JdbcTest
+@Sql("classpath:refreshTokenDDL.sql")
+class RefreshTokenRepositoryImplTest {
+
+    @Autowired
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+
+    private RefreshTokenRepositoryImpl refreshTokenRepositoryImpl;
+
+    @BeforeEach
+    void setUp() {
+        refreshTokenRepositoryImpl = new RefreshTokenRepositoryImpl(namedParameterJdbcTemplate);
+    }
```

Comment:
```suggestion
    private RefreshTokenRepositoryImpl refreshTokenRepositoryImpl;

    @Autowired
    RefreshTokenRepositoryImplTest(final NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
        this.refreshTokenRepositoryImpl = new RefreshTokenRepositoryImpl(namedParameterJdbcTemplate);
    }
```
BeforeEach로 매번 생성하는 것보다 이렇게 처리하는 건 어떨까요?

Reply:
반영하도록 하겠습니다.